dev-cpp/glog/glog-0.5.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 2011-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+CMAKE_ECLASS="cmake"
+
+inherit cmake-multilib
+
+if [[ "${PV}" == "9999" ]]; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/google/glog"
+fi
+
+DESCRIPTION="Google Logging library"
+HOMEPAGE="https://github.com/google/glog"
+if [[ "${PV}" == "9999" ]]; then
+	SRC_URI=""
+else
+	SRC_URI="https://github.com/google/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+fi
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc64 ~s390 ~x86 ~amd64-linux ~x86-linux"
+IUSE="gflags +libunwind llvm-libunwind test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="gflags? ( dev-cpp/gflags:0=[${MULTILIB_USEDEP}] )
+	libunwind? (
```

Comment:
Maybe we could deactivate `libunwind` support - `glog` is only used by `update-engine` I think ? 

Reading `glog` documentation[^1]:
> For that reason, if you use a 64-bit system and you need InstallFailureSignalHandler(), we strongly recommend you install libunwind before trying to configure or install google glog

Quick search on [`update-engine`](http://github.com/flatcar-linux/update-engine) repository does not show any sign of `InstallFailureSignalHandler`.

[^1]: https://github.com/google/glog#installation-notes-for-64-bit-linux-systems

Reply:
I think we are pulling in glog 0.4, which pulls libunwind unconditionally. 0.5 is still marked as unstable.